### PR TITLE
Northern Sami

### PR DESCRIPTION
**Pull request by Karamell**

### CODE REVIEW

File: src/locale/nb.js [hunk 1]
```diff
@@ -49,4 +49,3 @@ export default moment.defineLocale('nb', {
         doy : 4  // The week that contains Jan 4th is the first week of the year.
     }
 });
-
```
Comment (mj1856) [inline src/locale/nb.js]: This line removal looks extraneous.  Please rebase.  Thanks.

**Comment by Karamell** [comment]
No untouched file anymore @mj1856. No more `git commit -a` for me :-)

**Comment by lazreg87** [comment]
+1

**Comment by ichernev** [comment]
Thank you! I'll merge this in next release.

**Comment by ichernev** [comment]
Merged in d1332d4dda751fcfde1d1b76019afdec202e007a

**Pull request closed without merging**